src/Sulu/Bundle/TagBundle/Controller/TagController.php

```diff
@@ -148,9 +148,15 @@ public function cgetAction(Request $request)
             /** @var DoctrineListBuilderFactory $factory */
             $factory = $this->get('sulu_core.doctrine_list_builder_factory');
 
+            $fieldDescriptors = $this->getManager()->getFieldDescriptors();
             $listBuilder = $factory->create(self::$entityName);
 
-            $restHelper->initializeListBuilder($listBuilder, $this->getManager()->getFieldDescriptors());
+            $ids = array_filter(explode(',', $request->get('ids', '')));
+            if (count($ids) > 0) {
+                $listBuilder->in($fieldDescriptors['id'], $ids);
+            }
```

Comment:
i need that for the tags component of condition select where i only have ids but need the name f the tag